.github/workflows/ci.yml

```diff
@@ -51,10 +55,12 @@ jobs:
 
       - name: Create a release in Octopus
         uses: OctopusDeploy/create-release-action@v3
+        env: 
+          OCTOVERSION_ScheduledSemVer: ${{ env.OCTOVERSION_MajorMinorPatch }}-scheduledRun
         with:
           space: ${{ env.OCTOPUS_SPACE }}
           project: ${{ env.OCTOPUS_PROJECT }}
           git_ref: ${{ github.head_ref || github.ref_name }}
           git_commit: ${{ github.event.after || github.event.pull_request.head.sha }}
-          release_number: ${{ env.OCTOVERSION_FullSemVer }}
+          release_number: ${{ github.event_name == 'schedule' &&  env.OCTOVERSION_ScheduledSemVer || env.OCTOVERSION_FullSemVer }}
```

Comment:
🤔 

Does Server choose a channel based on package version or release version?

Would you consider driving a `channel:` input off the event, leaving it blank if not `schedule`?

Reply:
ah yes, the server chooses the channel based off the package version 

We could add a `channel:`  input , or add an extra step after nuke is run to set the package version environment variable as a conditional statement 

I think it would be clear to add it as an extra step, and let octopus decide the channel handling as opposed to overriding it